src/main/java/de/florianmichael/imguiexample/imgui/ImGuiImpl.java

```diff
@@ -87,7 +87,7 @@ public static void draw(final RenderInterface renderInterface) {
         ImGui.render();
         imGuiImplGl3.renderDrawData(ImGui.getDrawData());
 
-        GlStateManager._glBindFramebuffer(GL30.GL_FRAMEBUFFER, previousFramebuffer);
+        GlStateManager._glBindFramebuffer(GL30.GL_FRAMEBUFFER, 0);
```

Comment:
I don't think the line is needed at all, idk why I put it there. The previousFramebuffer int can also be inlined as it's the current framebuffer, not the previous one